charts/loki-simple-scalable/values.yaml

```diff
@@ -249,6 +249,8 @@ enterprise:
         uwsgi_temp_path       /tmp/uwsgi_temp;
         scgi_temp_path        /tmp/scgi_temp;
 
+        proxy_http_version    1.1;
```

Comment:
does this have any backwards compatibility issues?

Reply:
this affects the connections between nginx and the loki components. 

to my knowledge no backward compatiblity issues exists , HTTP1.1 has been out for a while :)

I _could_ put it behind a flag in helm but unless is the default value it will probably keep being used as 1.0 by most people ... which indeed is what i think is happening on nginx. The default has been left to 1.0 for legacy reason and people do reverse proxy using http 1.0 without even knowing.

Reply:
I will change all the prs ( i did one for each chart ) and set a flag in the values to enable proxy 1.1 with default value false , thinking about it i think is better.

Reply:
sounds good. please give me a mention when it's ready for another review :smile: